docs/content/PointerBox.md

```diff
@@ -72,12 +76,37 @@ function CaretSelector(props) {
 render(<PointerBoxDemo />)
 ```
 
-## System props
+## Props
+
+<PropsTable>
+  <PropsTableRow
+    name="caret"
+    type="top| top-left | top-right | right | right-top | right-bottom | bottom | bottom-left | bottom-right | left | left-top | left-bottom"
+    defaultValue="bottom"
```

Comment:
```suggestion
    defaultValue="'bottom'"
```

Reply:
Fixed. Thanks @colebemis. Is this g2g now?
<img width="528" alt="Screenshot 2022-01-24 at 21 47 00" src="https://user-images.githubusercontent.com/13340707/150870387-022be148-3fe2-4e31-925a-a4ced68c0594.png">